File: Practica4/suma_n_numeros.c
#include <stdio.h>
int suma_n_primeros_nums(int number);
void main () {
	int num, retry, resultado;
	for (retry = 1; retry >= 1; retry++) {
		printf("Escribe cuantos números quieres sumar: ");
		scanf(" %d", &num);
		if (num >= 1) {
			resultado = suma_n_primeros_nums(num);
			printf("La suma es: %d\n", resultado);
			retry = -1;
		}
		else{
			printf("Error, el número ha de ser mayor o igual a 1\n");
		}
	}
}

int suma_n_primeros_nums(int number) {
	int total, i;
	total = 0;
	i = 1;
	for (i = 0 ;i <= number; i++) {
		total = total + i;
	}
	return total;
}

File: Practica9/suma_n_numeros.py
retry = 1

while retry >= 1:
    valor = int(input("Escriba un valor."))
    if valor < 0:
        retry = retry + 1
    else:
        retry = 0
print(suma_n_primeros_nums(valor))

def suma_n_primeros_nums(valor):
	i = 0
	total = 0
	while i <= valor:
		total = total + i
		i = i + 1
	return total
